test/clj/cider/nrepl/middleware/info_test.clj

```diff
@@ -144,7 +144,7 @@
             (pr-str response))
         (is (= (:class response) "cider.nrepl.test.TestClass"))
         (is (= (:member response) "doSomething"))
-        (is (= (:arglists-str response) "[int int java.lang.String]"))
+        (is (= (:arglists-str response) "[a b c]"))
```

Comment:
Hmm, I wonder what changed this as it doesn't seem related to the previous PR.

Reply:
Well, this is a strange one. The tests do pass now in the REPL, but fail when run via `lein test`. Any ideas?